src/zope/traversing/api.py

```diff
@@ -122,17 +87,15 @@ def traverseName(obj, name, default=_marker, traversable=None, request=None):
 
 
 def getName(obj):
-    """Get the name an object was traversed via
+    """
+    Get the name an object was traversed via
     """
     return ILocationInfo(obj).getName()
 
 
 def getParent(obj):
-    """Returns the container the object was traversed via.
-
-    Returns None if the object is a containment root.
-    Raises TypeError if the object doesn't have enough context to get the
-    parent.
+    """
+    Returns the container the object was traversed via.
```

Comment:
~~Information about containment root and raised exceptions seems like it could be useful?~~

This seemed peculiar before I saw the interface `__doc__` copying at the end of the module.